IGNORED_LIBS

```diff
@@ -0,0 +1,11 @@
+libc.so.6
```

Comment:
I would put this as `define IGNORED_LIBS` inside the Makefile, since its usage is tied to it.

Reply:
Agree. I will change it out